wsgi/odyssy/events/models.py

```diff
@@ -9,13 +9,13 @@
 
 class Event(models.Model):
     name = models.CharField(max_length=30, default="Sample Name")
-    start_time = models.DateTimeField(default=datetime.datetime.now())
-    end_time = models.DateTimeField(default=datetime.datetime.now())
+    start_time = models.DateTimeField(default=datetime.datetime.now)
+    end_time = models.DateTimeField(default=datetime.datetime.now)
     description = models.TextField(max_length=300)
     place = models.CharField(max_length=50, null=True, default=None, blank=True)
 
     def __str__(self):
-        event_str = self.name  + " on " + str(self.start_time.day) + \
+        event_str = self.name + " on " + str(self.start_time.day) + \
             " - " + calendar.month_abbr[self.start_time.month] + \
             " - " + str(self.start_time.year)
         if self.place is not None:
```

Comment:
It is better if we make `""` as None. It might be helpful in future. We will not have to check for `None` as well as `""` in other places. What do you say @singhpratyush Sir?